src/components/FilterBar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { PropsType } from "../containers/FilterBarContainer";
+import * as objAssign from "object-assign";
+import { Store } from "azsearchstore";
+import { defaultCss } from "../utils/css";
+
+export type State = {};
+
+class FilterBar extends React.Component<PropsType, State> {
+  render() {
+    const { onClear, hasSelectedFacets } = this.props;
+    const text = "clear filter(s)";
+
+    return (
+      <div className="text-right">
+        { hasSelectedFacets ?
```

Comment:
nit: Not as important for simple componentes, but generally I find it easier to grok when conditional components are assigned to a variable. Makes the overall structure of html a little easier to reason about

for example:

```ts
let clearComponent = hasSelectedFacets ? <a> : ...;

return (
<div>
{clearComponent}
</div>
)
```